root/entity/Details.js

```diff
@@ -49,8 +49,12 @@ const WSLink = ({
   entityType === 'release' && inc.push('labels', 'discids', 'recordings');
```

Comment:
do you have a style guide that recommends this style for conditionals instead of an if statement? just curious - to me this is harder to read than
```js
if (entityType === 'release') {inc.push('labels', 'discids', 'recordings');}
```

Reply:
I don't think we have an eslint rule for that yet, but an `if` would look more clear to me too.

Reply:
I'd also prefer it. A rule doesn't seem to exist? But we could still try to find these somehow and fix them in one go at least, since we're unlikely to reintroduce them if nobody likes them :)

Reply:
Opened https://github.com/metabrainz/musicbrainz-server/pull/2341 for the cases I could find. There might be more (I did not check every `&&` by hand, just the ones followed by `<`)